acl_loader/main.py

```diff
@@ -409,10 +417,17 @@ def convert_l2(self, table_name, rule_idx, rule):
             else:
                 try:
                     rule_props["ETHER_TYPE"] = int(rule.l2.config.ethertype)
-                except:
-                    raise AclLoaderException("Failed to convert ethertype %s table %s rule %s" % (
+                except Exception:
+                    raise AclLoaderException("Failed to convert ethertype %s; table %s rule %s" % (
                         rule.l2.config.ethertype, table_name, rule_idx))
 
+        if rule.l2.config.vlan_id:
```

Comment:
>vlan_id [](start = 26, length = 7)

Do we need to validate input for:
1. vlan_id<=0
2. valn_id too large

Reply:
Orchagent does this, but I think it's probably a good idea to check here too (it'll be more obvious to the user!).

Reply:
done